types/three/src/math/Color.d.ts

```diff
@@ -255,7 +255,7 @@ export class Color {
      * Copies given color.
      * @param color Color to copy.
      */
-    copy(color: Color): this;
+    copy(color: this): this;
```

Comment:
Is there a benefit to changing the parameter to `this`? Seems like it could still just take a `Color` (and `Color4.copy()` could take a `Color4`)?

Reply:
The benefit was that it wouldn't need redefinition in Color4, but I neglected to remove the definition from Color4.

Reply:
Would you prefer me to make the change and remove the definition from color4 or define in both color and color4?

Reply:
I'm fine either way.